concent_api/common/decorators.py

```diff
@@ -336,3 +308,39 @@ def wrapper(*args, **kwargs):
             )
             raise
     return wrapper
+
+
+def get_response_from_view_with_db_retry(
```

Comment:
done, hope you will like it :)

Reply:
I tried to use `else`, but pylint says:
`Else clause on loop without a break statement (useless-else-on-loop)`. I think it is better too leave it like this instead of another one disable pylint. What do you think?